### PR TITLE
Mute

### PR DESCRIPTION
**Pull request by reidobandedo**

### CODE REVIEW

File: js/round.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 // https://github.com/algorithm0r/GamesProject/blob/Unicorn/game.js
 
 // Class Constants:
-var ROUND_TIME_LIMIT = 60; // 1 minute (in seconds)
+var ROUND_TIME_LIMIT = 10; // 1 minute (in seconds)
```
Comment (ddh) [inline js/round.js]: Remember to change this back right before you push to master

File: js/credits.js [hunk 2]
```diff
@@ -102,7 +99,11 @@ Credits.prototype.draw = function (ctx) {
     // Credits
     drawTextWithOutline(ctx, '48px Impact', 'Red Three Team', 240, 120, 'red', 'white', 5);
 
-
+    if (MUTED) {
```
Comment (ddh) [inline js/credits.js]: this piece of code gets repeated often; put it in gameengine in the draw method OR in the mute/unmuted functions in gameengine that you made.

File: js/endgame.js [hunk 3]
```diff
@@ -134,6 +134,12 @@ EndGame.prototype.draw = function (ctx) {
         // draw crown over winning goat
         this.crownAnimation.drawFrame(this.game.clockTick, ctx, winningGoat.x + 45, winningGoat.y - 30, 1);
     }
+
```
Comment (DuyH) [inline js/endgame.js]: this piece of code gets repeated often; put it in gameengine in the draw method

Comment (ddh) [inline js/endgame.js]: DRY

File: js/round.js [hunk 4]
```diff
@@ -232,12 +233,18 @@ Round.prototype.draw = function (ctx) {
     this.drawTimer(ctx);
     drawTextWithOutline(ctx, "32px Impact", "Round #" + (ROUNDS_PLAYED + 1), 650, 40, 'purple', 'white');
     drawTextWithOutline(ctx, "32px Impact", "Oh My Goat!", 20, 40, 'purple', 'white');
+
```
Comment (ddh) [inline js/round.js]: DRY

File: js/scoreboard.js [hunk 5]
```diff
@@ -51,9 +51,17 @@ Scoreboard.prototype.draw = function (ctx) {
     drawTextWithOutline(ctx, "40px Impact", this.goats[1].score, 310, 202, this.goats[1].color, 'white'); // winner #2
     drawTextWithOutline(ctx, "40px Impact", this.goats[2].score, 585, 288, this.goats[2].color, 'white'); // winner #3
     drawTextWithOutline(ctx, "40px Impact", this.goats[3].score, 220, 368, this.goats[3].color, 'white'); // winner #4
+
```
Comment (ddh) [inline js/scoreboard.js]: DRY

File: js/title.js [hunk 6]
```diff
@@ -133,6 +134,11 @@ Title.prototype.draw = function (ctx) {
             "rgba(255, 255, 255, 0.4)",
             "rgba(255, 255, 255, 0)");
     }
+    if (MUTED) {
```
Comment (ddh) [inline js/title.js]: DRY

File: js/tutorial.js [hunk 7]
```diff
@@ -170,6 +170,12 @@ Tutorial.prototype.draw = function (ctx) {
                              'rgba(255, 255, 255, 0.4)',
                              'rgba(255, 255, 255, 0)');
     }
+
```
Comment (ddh) [inline js/tutorial.js]: DRY

**Pull request merged**